crypto/poly1305/poly1305_ameth.c

```diff
@@ -0,0 +1,121 @@
+/*
+ * Copyright 2007-2016 The OpenSSL Project Authors. All Rights Reserved.
+ *
+ * Licensed under the OpenSSL license (the "License").  You may not use
+ * this file except in compliance with the License.  You can obtain a copy
+ * in the file LICENSE in the source distribution or at
+ * https://www.openssl.org/source/license.html
+ */
+
+#include <stdio.h>
+#include "internal/cryptlib.h"
+#include <openssl/evp.h>
+#include "internal/asn1_int.h"
+#include "internal/poly1305.h"
+#include "poly1305_local.h"
+
+#define POLY1305_TEST_PRIVATE_KEY_FORMAT
```

Comment:
This define, and the corresponding ifdef/else/endif were taken from HMAC, and I'm not sure it really belongs here. I can remove it if necessary.